nautobot/dcim/views.py

```diff
@@ -1931,7 +1931,6 @@ def get_extra_context(self, request, instance):
                 "tags",
             )
             .select_related("lag", "cable")
-            .order_by("_name")
```

Comment:
What was incorrect about this?

Reply:
There's something weird going on with our `_name` field. I'll try to see if I can write a test for it and fix it.

```
>>> for i in modular_device.all_interfaces.values("name", "_name"):
...     print(i["name"])
...     print(i["_name"])
... 
mgmt0
9999999999999999mgmt000000............
TenGigabitEthernet1/2/1
0001000299999999TenGigabitEthernet000001............
TenGigabitEthernet1/2/2
0001999999999999TenGigabitEthernet..................{module}/00000002
TenGigabitEthernet1/2/3
0001999999999999TenGigabitEthernet..................{module}/00000003
TenGigabitEthernet1/2/4
0001999999999999TenGigabitEthernet..................{module}/00000004
Ethernet1/4/1
0001000499999999Ethernet000001............
Ethernet1/4/2
0001999999999999Ethernet..................{module}/00000002
Ethernet1/5/1
0001999999999999Ethernet..................{module}/00000001
Ethernet1/5/2
0001999999999999Ethernet..................{module}/00000002
>>> for i in modular_device.all_interfaces.all():
...     i.save()
... 
>>> for i in modular_device.all_interfaces.values("name", "_name"):
...     print(i["name"])
...     print(i["_name"])
... 
mgmt0
9999999999999999mgmt000000............
TenGigabitEthernet1/2/1
0001000299999999TenGigabitEthernet000001............
TenGigabitEthernet1/2/2
0001000299999999TenGigabitEthernet000002............
TenGigabitEthernet1/2/3
0001000299999999TenGigabitEthernet000003............
TenGigabitEthernet1/2/4
0001000299999999TenGigabitEthernet000004............
Ethernet1/4/1
0001000499999999Ethernet000001............
Ethernet1/4/2
0001000499999999Ethernet000002............
Ethernet1/5/1
0001000599999999Ethernet000001............
Ethernet1/5/2
0001000599999999Ethernet000002............
```

Reply:
Ew, something where it's not liking the `{module}` syntax huh?

Reply:
Looks to me like I optimized the model save a little bit too much [here](https://github.com/nautobot/nautobot/blob/cd6bff67514b148293fe37c7fa1594e29d03f115/nautobot/dcim/models/device_components.py#L182)